akka-projection-core/src/main/scala/akka/projection/SourceProvider.scala

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection
```

Comment:
would be good if we placed things in scaladsl package if we know that it will be needed. This must be scaladsl/javadsl because of different Source and Option